exercises/practice/list-ops/.meta/config.json

```
@@ -0,0 +1,18 @@
+{
+  "blurb": "Implement basic list operations.",
+  "authors": [
+    "fap"
```

Note that this is the user's github username, not the exercism handle:

```suggestion
    "fapdash"
```

---

Done